middleware/errorHandler.js

```diff
@@ -0,0 +1,7 @@
+module.exports = (err, req, res, next) => {
+  return res.status(err.status || 500).json({
+    error: {
+      message: error.message || 'Oops! Something went wrong',
```

Comment:
`error` doesn't exist here, I think you meant to use your `err` parameter.

```suggestion
      message: err.message || 'Oops! Something went wrong',
```